src/common/Card/helper.ts

```diff
@@ -0,0 +1,41 @@
+// just a list of all months in order
+const months = [
+  "January",
+  "February",
+  "March",
+  "April",
+  "May",
+  "June",
+  "July",
+  "August",
+  "September",
+  "October",
+  "November",
+  "December",
+];
+
+// Input: Date object, duration (minutes)
+// Output: Formatted string for start time and end time of an event
+export const format_time = (time: Date, duration: number): string => {
```

Comment:
Is there a specific reason this function has a snake_case naming? It might be better if we stuck to camelCase for all functions, even in helpers

Reply:
nope, fixed it.